components/compass-runtime-agent/internal/compassconnection/package_test.go

```diff
@@ -68,7 +72,7 @@ var (
 			}
 		}
 		// only check if x-request-id header was set
-		return input[RequestIDHeader] != ""
+		return true
```

Comment:
We should remove RequestIDHeader processing from that iteration. It is not used anymore for request identification. This can be also simplified, since we use only one header in the request `ConnectorTokenHeader`.

Reply:
Done ✅